dist/src/main/xslt/standalone.xsl

```diff
@@ -80,6 +85,17 @@
     </cache-container>
   </xsl:template>
 
+  <!-- Add MDB support for Hawkular Accounts -->
```

Comment:
Note that this is only for our dist. We don't need ActiveMQ, so, it's easier to just enable the JMS and MDB support on the Wildfly we already have. So, this won't have similar changes in the main Hawkular build.